style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  background-image: url('images/Header bg.png');
+}
+
+.navigator {
+  display: flex;
+  background-color: white;
+  justify-content: flex-end;
+  margin-right: 10px;
+  padding: 10px;
+}
```

Comment:
- [ ] Kindly add this property to navigator class `align-items: center;` to make your header align in the center.